### DISCUSSION TITLE
Grouped Stacked Columns 100 percentage not working

### DISCUSSION THREAD

**Discussion by tusharsalave** [apexcharts/apexcharts.js#4536] [General]
I want to create a 100% stacked column chart for grouped data. The 100% chart works for a single column but not for grouped columns. i want to create a graph like this
<img width="460" alt="Screenshot 2024-06-20 at 12 38 02 PM" src="https://github.com/apexcharts/apexcharts.js/assets/54859747/c1c42bfa-a0b0-44c7-b2a7-fb1552ad5bac">

**Comment by jpscohort** [comment]
I have been trying to do the same thing. This works for non percentage stacked grouped columns, but not the percentage one.
Grouped percentage calculates the percentages for all groups and not group by group.
